test/conftest.py

```diff
@@ -1,4 +1,4 @@
-from common_utils import IN_CIRCLE_CI, CIRCLECI_GPU_NO_CUDA_MSG, IN_FBCODE, IN_RE_WORKER, CUDA_NOT_AVAILABLE_MSG
+from test.common_utils import IN_CIRCLE_CI, CIRCLECI_GPU_NO_CUDA_MSG, IN_FBCODE, IN_RE_WORKER, CUDA_NOT_AVAILABLE_MSG
```

Comment:
Shouldn't all imports withing the `test` folder be relative imports?

Reply:
Do we have a rule for when to use relative vs. absolute imports I should follow? Otherwise it makes no difference. Can change if you like.

Reply:
I've always used only-relative imports for imports within a package... I don't even remember why.
If anything, it should make the import sorting more straightforward?